intranet/apps/emailfwd/management/commands/get_senior_forwards.py

```diff
@@ -0,0 +1,12 @@
+from django.core.management.base import BaseCommand
+
+from ....emailfwd.models import SeniorEmailForward
+
+
+class Command(BaseCommand):
+    help = "Gets all senior email forwards"
+
+    def handle(self, *args, **options):
+        forwards = [f for f in SeniorEmailForward.objects.all() if f.user.is_senior]
```

Comment:
```suggestion
        forwards = SeniorEmailForward.objects.filter(user__is_senior=True)
```

Reply:
Maybe it's just me, but when I use that I get `django.core.exceptions.FieldError: Related Field got invalid lookup: is_senior` - my understanding is that Django can't `filter` by properties like `is_senior` since they're not actually in SQL, which is why I used a list comprehension instead.

Reply:
Ah no you're right -- `is_senior` isn't a database field. You could do `graduation_year`=`get_senior_graduation_year()`, which *should* be *slightly* more memory efficient, but either way is probably fine.